### DISCUSSION TITLE
BLCHeader meaning and usage

### DISCUSSION THREAD

**Discussion by chayan-blip** [ben-manes/caffeine#1010] [General]
Hi Ben,
Can you please help me to understand the motivation behind the BLCHeader and PadDrain codes.
I went through the following  [link](https://stackoverflow.com/questions/68706584/java-byte-i-cant-understand-the-usage-here)
As per a google search this may be related to False sharing
 [File](https://github.com/ben-manes/caffeine/blob/9d5e6de640fdd6a7050f940a3ef15c77545b5559/caffeine/src/main/java/com/github/benmanes/caffeine/cache/BoundedLocalCache.java)

**Comment by ben-manes** [comment]
The first link is correct, this is for false sharing. The format comes from this [article](https://shipilev.net/jvm/objects-inside-out/#_observation_hierarchy_tower_padding_trick_collapse_in_jdk_15). Does that help answer your questions?
